lib/private/BinaryFinder.php

```diff
@@ -37,15 +50,10 @@ public function findBinaryPath(string $program) {
 		if (\OCP\Util::isFunctionEnabled('exec')) {
 			$exeSniffer = new ExecutableFinder();
 			// Returns null if nothing is found
-			$result = $exeSniffer->find($program, null, [
-				'/usr/local/sbin',
-				'/usr/local/bin',
-				'/usr/sbin',
-				'/usr/bin',
-				'/sbin',
-				'/bin',
-				'/opt/bin',
-			]);
+			$result = $exeSniffer->find(
+				$program,
+				null,
+				$this->config->getSystemValue('binary_search_paths', self::DEFAULT_BINARY_SEARCH_PATHS));
 			if ($result === null) {
 				$result = false;
 			}
```

Comment:
```suggestion
				false,
				$this->config->getSystemValue('binary_search_paths', self::DEFAULT_BINARY_SEARCH_PATHS));
```

An opportunity to streamline. :-)

Reply:
Can't be applied because false is not a valid default value